java/src/main/java/ai/rapids/cudf/Aggregation.java

```diff
@@ -62,15 +62,16 @@ enum Kind {
         LAG(23),
         PTX(24),
         CUDA(25),
-        M2(26),
-        MERGE_M2(27),
-        RANK(28),
-        DENSE_RANK(29),
-        PERCENT_RANK(30),
-        TDIGEST(31), // This can take a delta argument for accuracy level
-        MERGE_TDIGEST(32), // This can take a delta argument for accuracy level
-        HISTOGRAM(33),
-        MERGE_HISTOGRAM(34);
+        HOST_UDF(26),
```

Comment:
Append `HOST_UDF` to the tail? then do not touch other aggregation values.

Reply:
Unfortunately `HOST_UDF` is classified as "UDF" which should be grouped together with other UDF kinds. That's why I have to insert it in the middle.